kotlin-result-coroutines/src/commonTest/kotlin/com.github.michaelbull.result.coroutines/binding/SuspendableBindingTest.kt

```diff
@@ -25,7 +25,7 @@ class SuspendableBindingTest {
             return Ok(2)
         }
 
-        runBlockingTest {
+        return runBlockingTest {
```

Comment:
Is this necessary if we have a `return` on line 52? (also applies to other added `return` statements)

Reply:
Unfortunately yes. Without it the tests would never fail with Kotlin/JS. Instead, an `UnhandledPromiseRejectionWarning` is being printed to the console (see https://youtrack.jetbrains.com/issue/KT-22228#focus=Comments-27-3964713.0-0).

Reply:
That's disappointing